core/src/main/java/io/questdb/cutlass/http/HttpResponseSink.java

```diff
@@ -774,7 +772,7 @@ public void sendStatus(int code, CharSequence message, CharSequence header, Char
 
         public void sendStatus(int code) throws PeerDisconnectedException, PeerIsSlowToReadException {
             buffer.clearAndPrepareToWriteToBuffer();
-            headerImpl.status(httpVersion, code, CONTENT_TYPE_HTML, -2L);
+            headerImpl.status(httpVersion, code, null, -2L);
```

Comment:
we need some comments to explain why content type became null and how it we send content type to the client if not here

Reply:
`sendStatus()` is used to send status codes without content like 204 or 304.
Sending content type without content is wasteful so we will send 

```
String expectedResponseHeader2 = "HTTP/1.1 304 Not Modified\r\n" +
        "Server: questDB/1.0\r\n" +
        "Date: Thu, 1 Jan 1970 00:00:00 GMT\r\n" +
        "\r\n";
```

instead of 

```
String expectedResponseHeader2 = "HTTP/1.1 304 Not Modified\r\n" +
        "Server: questDB/1.0\r\n" +
        "Date: Thu, 1 Jan 1970 00:00:00 GMT\r\n" +
        "Content-Type: text/html; charset=utf-8\r\n" +
        "\r\n";
```